pkg/networkutils/network.go

```diff
@@ -518,6 +518,32 @@ func setupENINetwork(eniIP string, eniMAC string, eniTable int, eniSubnetCIDR st
 		return errors.Wrapf(err, "eni network setup: failed to define gateway address from %v", ipnet.IP)
 	}
 
+	// Explicitly set the IP on the device if not already set.
+	// Required for older kernels.
+	// ip addr show
+	// ip add del <eniIP> dev <link> (if necessary)
+	// ip add add <eniIP> dev <link>
+	log.Debugf("Setting up ENI's primary IP %s", eniIP)
+	addrs, err := netlink.AddrList(link, unix.AF_INET)
```

Comment:
`s/netlink/netLink/g` -  (Please replace `l` with `L`.)
Then, `netLink.AddrList()` calls wrapper library and `mockNetLink.EXPECT().AddrList` in unit test will be called as you expected.

Reply:
replace  with
```
 addrs, err := netLink.AddrList(link, unix.AF_INET),
```
unit-test should pass